app/components/primer/alpha/layout.rb

```diff
@@ -14,6 +14,8 @@ module Alpha
     #   Keyboard navigation follows the markup order. Decide carefully how the focus order should be be by deciding whether
     #   `main` or `sidebar` comes first in code. The code order won’t affect the visual position.
     class Layout < Primer::Component
+      status :alpha
```

Comment:
this was missing, and i think it was causing issues with the docs being rendered

Reply:
Could we add a test to catch this next time?